project/frontend/src/scripts/recommendation_script.js

```diff
@@ -72,6 +73,8 @@ function makePromisesArray(preferences) {
     searchParams.append('location', currLocation.lat + ',' + currLocation.lng);
     if (radius.pref) {
       searchParams.append('radius', milesToMeters(radius.pref));
+    } else {
+      searchParams.append('radius', defaultRadius)
```

Comment:
so if they don't put a radius in results come up with a 50,000 mile radius?

Reply:
no this is in meters

Reply:
if you put a location you have to specify a radius (I didn't have a radius by default before that's why I added this), so if the user doesn't choose a specific distance it defaults to the maximum radius the API allows (see "Optional Parameters" here https://developers.google.com/places/web-service/search#TextSearchRequests)

Reply:
gotcha! is there a reason you chose to do the maximum radius as opposed to a value somewhere in the middle?

Reply:
Yeah just because it's only ~30 miles and I didn't think it made sense to be limiting the restaurants that are returned if it's not a field that the user specified a preference for.

Reply:
gotcha! Do we have consideration for if the user wants delivery? I'm not sure if this is handled somewhere else in the algorithm or API but I don't think it's typical for restaurants to deliver 30 miles away so maybe this could be a special case where the default radius would be something different / smaller!